src/Microsoft.DotNet.Interactive/KernelScheduler.cs

```diff
@@ -135,7 +132,8 @@ private void Run(ScheduledOperation operation)
     {
         _currentTopLevelOperation ??= operation.Value;
 
-        using var logOp = Log.OnEnterAndConfirmOnExit($"Run: {operation.Value}");
+        using var logOp = Log.OnEnterAndConfirmOnExit();
+        logOp.Info("{0}", operation.Value);
```

Comment:
It would be great if we could say `logOp.Infi(operation.Value)` instead. This would need a change in PocketLogger to make the message template string optional.

Reply:
Adding ability to pass argument(s) in `OnEnterAndConfrimOnExit` would also make the above case simpler. We could then just say `Log.OnEnterAndConfirmOnExit(operation.Value)`.